File: Problem10/code.js
let checkPrime = (n) =>{

    let result = true

    for(let y = 2; y <= Math.sqrt(n); y++){

        if(n % y === 0){
            result = false;
            break;
        }

    }

    return result
}


let findPrimeSum = (num) =>{


    let arrayOfPrimes = []

    for(let x = 2; x < num; x++){

        if(checkPrime(x)){
            arrayOfPrimes.push(x)
        }
    }

    let sumOfPrimes = arrayOfPrimes.reduce((a, b) => a + b, 0)

    return sumOfPrimes

}

console.log(findPrimeSum(2000000))

File: Problem 10/code.py
import math

def check_prime(n):
    result = True
    
    for x in range(2, int(math.ceil(math.sqrt(n+1)))):
        if n % x == 0:
            result = False
            break
        
    return result

def find_prime_sum(num):
    list_of_primes = []
    
    for y in range(2, num):
        if check_prime(y):
            list_of_primes.append(y)
    return sum(list_of_primes)

print(find_prime_sum(2000000))